### PR TITLE
feat(rpc): generate library

### PR DESCRIPTION
**Pull request by coryan**

### CODE REVIEW

**Comment by coryan** [comment]
I will refactor `gax` to use this library in a followup PR.

File: src/generated/rpc/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Google Cloud SDK for Rust - Google RPC Types
+
+Defines RPC types.
```
Comment (dbolduc) [inline src/generated/rpc/README.md:3]: Aside: is this generated and maintained by sidekick? If so, should it have a comment saying so?

File: src/generated/rpc/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+version              = "0.1.0-rc2"
+description          = "Google Cloud SDK for Rust - Google RPC Types"
+edition.workspace    = true
+authors.workspace    = true
+license.workspace    = true
+repository.workspace = true
+keywords.workspace   = true
+categories.workspace = true
+
+[dependencies]
```
Comment (dbolduc) [inline src/generated/rpc/Cargo.toml:28]: not saying it is worth doing, but seems like the dependencies could be pruned for packages without services

**Review by dbolduc — APPROVED** [review]

File: src/generated/rpc/Cargo.toml (hunk 2 above)
Comment (coryan) [reply src/generated/rpc/Cargo.toml:28]: I think it is worth doing #492

File: src/generated/rpc/README.md (hunk 1 above)
Comment (coryan) [reply src/generated/rpc/README.md:3]: #493

**Pull request merged**